core/src/main/scala/quasar/destination/avalanche/AvalancheDestinationModule.scala

```diff
@@ -103,13 +103,13 @@ object AvalancheDestinationModule extends DestinationModule {
       Sync[F].delay(
         Executors.newFixedThreadPool(
           threadCount,
-          NamedDaemonThreadFactory(name))))(es => Sync[F].delay(es.shutdown()))
+          qc.NamedDaemonThreadFactory(name))))(es => Sync[F].delay(es.shutdown()))
       .map(ExecutionContext.fromExecutor(_))
 
-  private def unboundedPool[F[_]: Sync](name: String): Resource[F, ExecutionContext] =
+  private def unboundedPool[F[_]: Sync](name: String): Resource[F, Blocker] =
```

Comment:
We should eventually move this to `quasar.`. It's used in pretty much every JDBC destination.